### PR TITLE
Add sig-migrate's reviewers

### PR DESCRIPTION
**Pull request by overvenus**
As title.

### CODE REVIEW

**Review by BusyJay — APPROVED** [review]
Thanks for your contributions!

**Comment by tisonkun** [comment]
/hold

@overvenus please also update the membership file.

I'll try to add their contributions on reviewing later...

**Review by skyzh — APPROVED** [review]
+1, Congrats!

**Comment by tisonkun** [comment]
> /hold
> 
> @overvenus please also update the membership file.
> 
> I'll try to add their contributions on reviewing later...

Done and updated.

File: votes/0147-3pointer-and-glorv-as-tikv-reviewers.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+I (@overvenus) hereby nominate @3pointer and @glorv as TiKV Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 72 hours unless there is an objection or not enough votes.
+
+## Result
+
+Approved by X binding votes, X non-binding votes.
```
Comment (tisonkun) [inline votes/0147-3pointer-and-glorv-as-tikv-reviewers.md:23]:
```suggestion
Approved by 2 binding votes, 2 non-binding votes.
```

72 hours exceeded. @overvenus you may ask a maintainer to conclude the result and accept the vote by merging it.

Comment (tisonkun) [reply votes/0147-3pointer-and-glorv-as-tikv-reviewers.md:23]: /unhold

**Pull request merged**